content/docs/studio/index.md

```diff
@@ -1,3 +1,12 @@
+<admon>
+
+**We have renamed Views to Projects in Iterative Studio.**
+
+Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
+settings_ are now called _Project settings_; and so on.
+
+</admon>
+
```

Comment:
This page doesn't mention _Projects_ (ex _Views_) so I don't see need for this very prominent admonition. In fact it's lacking all context and thus probably confusing.

```suggestion
<admon>

**We have renamed Views to Projects in Iterative Studio.**

Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
settings_ are now called _Project settings_; and so on.

</admon>

```

Reply:
Removed.